lte/gateway/c/core/oai/tasks/nas/emm/ServiceRequestHdl.c

```diff
@@ -237,6 +229,21 @@ status_code_e emm_recv_initial_ext_service_request(
     OAILOG_FUNC_RETURN(LOG_NAS_EMM, RETURNerror);
```

Comment:
Can we modify the above log message from: 
"No EMM context exists for the UE (ue_id=" MME_UE_S1AP_ID_FMT ") \n",
to

"EMM-PROC- Extended Service Request received but no EMM context exists for the UE (ue_id=" MME_UE_S1AP_ID_FMT ") \n",